infra/resources/_modules/function_apps/locals.tf

```diff
@@ -76,6 +76,17 @@ locals {
       FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
       FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
       FETCH_KEEPALIVE_TIMEOUT             = "60000"
-    })
+
+      CosmosDbEndpoint     = var.cosmos_db_endpoint
+      CosmosDbDatabaseName = var.cosmos_database_names[0]
+
+      PdvTokenizerApiBaseURL = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerTestUUID   = "c13b2aec-1597-4abd-a735-aacf2f935c93"
```

Comment:
Is it not possible to take the values ​​already defined for the other Function via Terraform Variable?
In order to avoid duplicate https://github.com/pagopa/io-wallet/blob/8fae4c133153ec05db6706a522dbe14f4adb6dc6/infra/resources/_modules/function_apps/locals.tf#L38

CC @Krusty93

Reply:
yes right